packages/@aws-cdk/aws-ec2/lib/instance-types.ts

```diff
@@ -1171,6 +1182,8 @@ export class InstanceType {
       [InstanceClass.P3]: 'p3',
       [InstanceClass.PARALLEL3_NVME_DRIVE_HIGH_PERFORMANCE]: 'p3dn',
       [InstanceClass.P3DN]: 'p3dn',
+      [InstanceClass.PARALLEL4_NVME_DRIVE_EXTENDED]: 'p4dE',
```

Comment:
```suggestion
      [InstanceClass.PARALLEL4_NVME_DRIVE_EXTENDED]: 'p4de',
```

should the `E` be lowercase?

Reply:
Thanks for the hint. I corrected the value.